tests/integration/base.py

```diff
@@ -124,22 +124,96 @@ def _create_session(cursor, session_id, user_id):
 
 def _create_global_topics(cursor):
     # type: (psycopg2.extensions.cursor) -> None
+
     cursor.execute(
-        "insert into"
-        " phpbb_topics(topic_id, forum_id, topic_title, topic_time,"
-        "  topic_first_poster_name, topic_first_post_id, topic_type)"
-        " values (0,0,'naslov_teme_0',10,'ime',0,3),"
-        " (1,0,'naslov teme 1',13,'drugi poster',1,3),"
-        " (2,0,'naslov teme 2',200,'post it',2,3),"
-        " (3,0,'naslov teme 3',256,'posted it',3,3)"
+        "insert into phpbb_topics ("
+        "topic_id"
+        ",forum_id"
+        ",topic_title"
+        ",topic_time"
+        ",topic_first_poster_name"
+        ",topic_first_post_id"
+        ",topic_type"
+        ") values ("
```

Comment:
Whitespace in strings are missing (simulating you would be writing into a file).

Also, I see you could congest each row into one line. While keeping field list with each column in its own line (for readability).

Example:
```
        "insert into phpbb_topics ("
        "  topic_id"
        "  ,forum_id"
        "  ,topic_title"
        "  ,topic_time"
        "  ,topic_first_poster_name"
        "  ,topic_first_post_id"
        "  ,topic_type"
        ") values"
        "(0,0,'topic title 0',10",'name'",0,3)"
        ", (1, 0, topic title 1', 13, 'second poster', 1, 3)"
        ", (2, 0, 'topic title 2', 200, 'post it', 2, 3)"
        ...
```